src/argilla_llama_index/helpers.py

```diff
@@ -41,3 +40,48 @@ def _calc_time(events_data: Dict[str, List[CBEvent]], id: str) -> float:
     start_time = events_data[id][0].time
     end_time = events_data[id][1].time
     return _get_time_diff(start_time, end_time)
+
+def _create_svg(data: List) -> str:
```

Comment:
might we simplify this easily? Don't spent too much time but I was wondering. Also, can you add a dedicated PR to include this in the docs. community/integrations or tutorials/integrations